CSharpFunctionalExtensions/Result/Extensions/CombineAsyncLeft.cs

```diff
@@ -4,9 +4,9 @@
 
 namespace CSharpFunctionalExtensions
```

Comment:
you can remove the platform divergence in functions using: 

```
#if NET40
using Task = System.Threading.Tasks.TaskEx;
#else
using Task = System.Threading.Tasks.Task;
#endif
```
![image](https://user-images.githubusercontent.com/22896741/64185040-1c220d80-ce75-11e9-9b0a-999c8c56c493.png)